solarforecastarbiter/plotting/timeseries.py

```diff
@@ -274,7 +279,7 @@ def generate_forecast_figure(forecast, data):
 
 
 @to_components
-def generate_observation_figure(observation, data):
+def generate_observation_figure(observation, data, limit=pd.Timedelta('3d')):
```

Comment:
Ok to leave it alone, but why do you have a different limit here than for the forecast?

Reply:
just maintaining how it was originally. And since forecast time steps are currently 1hr, shouldn't matter for the demo. We should add a way to move both forecasts and observations in time though